### DISCUSSION TITLE
Vlad A. Ionescu - Earthly

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#215] [Announcements]
- https://earthly.dev
- https://earthly.dev/blog/shutting-down-earthly-ci/
- https://twitter.com/VladAIonescu
- https://vladaionescu.com

## Questions

- [x] What is earthly? How does it make by builds consistent?
- [x] What was earthly Ci? Was the chain of events that led you to shut it down so quickly?
- [x] Why did you name it earthly? 
- [x] What are satellite's how are they an equivalent replacement?
- [x] What problem were you originally trying to solve when you decided to build this?
- [x] What are things earthly does to make builds faster?
- [x] How does the build caching work without any config?
- [x] Does it work with monorepo? Both JS and multi lang?
- [x] What the the DX of earthly look like? What tools does it replace?
- [x] What platforms are supported? How does it integrate with my CI?

###  Future

- [x] What's next? 
- [x] What's your spiciest dev take?
- [x] What do you think the future of builds is? To me it seems we do too much,

**Comment by just-be-dev** [comment]
Tooltips:
- https://www.burnerapp.com/
- https://www.bhoite.com/sculptures/boron-lander/

**Comment by hipstersmoothie** [comment]
- https://docs.axflow.dev/guides/models/getting-started.html
- https://www.hyperdx.io/

**Comment by vladaionescu** [comment]
Tooltip:

* https://github.com/moby/buildkit
